contracts/Verifier.sol

```diff
@@ -52,23 +55,23 @@ contract Verifier {
 
         //ClaimManager constraints don't allow for sessionId 0
         uint sessionId = sessionsCount+1;
-        sessions[sessionId] = VerificationSession({
-            id: sessionId,
-            claimant: claimant,
-            challenger: challenger,
-            input: _input,
-            output: _output,
-            lastClaimantMessage: now,
-            lastChallengerMessage: now,
-            lowStep: 0,
-            lowHash: keccak256(_input),
-            medStep: 0,
-            medHash: 0,
-            highStep: steps,
-            highHash: keccak256(_output)
-        });
+        VerificationSession storage s = sessions[sessionId];
```

Comment:
is there a reason we changed the format of this?

Reply:
I tried to work around stack limitation, but it didn't work out.

Reply:
makes sense, thanks